conf/modules.config

```diff
@@ -34,6 +34,11 @@ process {
         ]
     }
 
+    withName: SOURMASH_COMPARE {
+        ext.args = "-k 21"
```

Comment:
I would probably do 31 as the default! 21 is better when we expect evolutionary divergence between the things we're comparing, and since these are assemblies/reads from the same sample, 31 is probably a better default. although honestly they would probably return highly similar results

Reply:
updated